packages/core/EventManager.ts

```diff
@@ -489,6 +489,22 @@ export default class EventManager {
    */
   private async createAllCalendarEvents(event: CalendarEvent) {
     let createdEvents: EventResult<NewCalendarEventType>[] = [];
+
+    const fallbackToFirstConnectedCalendar = async () => {
+      /**
+       *  Not ideal but, if we don't find a destination calendar,
+       *  fallback to the first connected calendar - Shouldn't be a CRM calendar
+       */
+      const [credential] = this.calendarCredentials.filter((cred) => !cred.type.endsWith("other_calendar"));
+      if (credential) {
+        const createdEvent = await createEvent(credential, event);
+        log.silly("Created Calendar event", safeStringify({ createdEvent }));
+        if (createdEvent) {
+          createdEvents.push(createdEvent);
+        }
+      }
+    };
```

Comment:
Reusing this logic so abstracted into it's own function within `createAllCalendarEvents`